### DISCUSSION TITLE
monai bundles not found

### DISCUSSION THREAD

**Discussion by qwang0225** [Project-MONAI/MONAI#7277] [Q&A] [answered]
Hi, I was trying to download monai bundles but keep getting errors. I tried different models and made sure the name was correct. Can someone take a look? Thanks. 
`from monai.bundle import ConfigParser, download`
```
data_dir = r"C:\Users\frank\projects\whole_body CT segmentation"
model_name = "wholeBrainSeg_Large_UNEST_segmentation"
download(name=model_name, bundle_dir=data_dir)
```
```
HTTPError: 404 Client Error: Not Found for url: https://api.ngc.nvidia.com/v2/models/nvidia/monaihosting/wholeBrainSeg_Large_UNEST_segmentation
```

**Comment by KumoLiu** [comment]
Fixed in https://github.com/Project-MONAI/MONAI/pull/7280.

**Reply by qwang0225** [reply]
I just tried to download. It still did not work for either` wholeBrainSeg_Large_UNEST_segmentation` or `wholeBody_ct_segmentation`.  Maybe the way I tried to download was not correct.

**Reply by KumoLiu** [reply]
Did you try with the dev branch?

**Reply by qwang0225** [reply]
I'm not sure how to use dev branch.  I just did this.
```
model_name = "wholeBrainSeg_Large_UNEST_segmentation"
monai.bundle.download(name=model_name, bundle_dir=data_dir)
```

**Reply by KumoLiu** [reply, marked as answer]
Hi @qwang0225, the latest change is on the dev branch, you may update it on your own or wait for the weekly release.
https://pypi.org/project/monai-weekly/#history
https://docs.monai.io/en/stable/installation.html#from-pypi
